### PR TITLE
🔖 Release hub 0.30

### PR DESCRIPTION
**Pull request by chaichontat**

### CODE REVIEW

File: docs/changelog/2024.md [hunk 1]
```diff
@@ -16,6 +16,23 @@
 .. role:: small
 ```
 
+## 2024-11-13 {small}`hub 0.30.1`
```
Comment (falexwolf) [inline docs/changelog/2024.md:19]: Let's just use 0.30 here. The patch number doesn't make much sense yet.

File: docs/changelog/2024.md [hunk 2]
```diff
@@ -16,6 +16,23 @@
...
+
+### Enhancements
```
Comment (falexwolf) [inline docs/changelog/2024.md:21]: Please take a brief look at the right sidebar and whether it helps to have these as subheadings or whether a simple paragraph (possibly boldened) looks better at least for the 2024 page where no other release has subheadings.

File: docs/changelog/2024.md [hunk 3]
```diff
@@ -64,6 +81,50 @@ More changes:
 - 🔥 Remove `.list_source()` [PR](https://github.com/laminlabs/bionty/pull/130) [@sunnyosun](https://github.com/sunnyosun)
 - 🎨 Remove update in add_ontology_from_df [PR](https://github.com/laminlabs/bionty/pull/129) [@sunnyosun](https://github.com/sunnyosun)
 
+## 2024-10-14 {small}`hub 0.29.0`
+
+### Enhancements
+
+#### Improve filtering
```
Comment (falexwolf) [inline docs/changelog/2024.md:88]: Definitely not 4 levels of headings unless we put extensive effort into styling it.

Not in 2024.

File: docs/changelog/2024.md (hunk 1 above)
Comment (chaichontat) [reply docs/changelog/2024.md:19]: Used it to test if laminci failure was due to an existing tag.

File: docs/changelog/2024.md (hunk 3 above)
Comment (chaichontat) [reply docs/changelog/2024.md:88]: Fixed.

File: docs/changelog/2024.md (hunk 2 above)
Comment (falexwolf) [reply docs/changelog/2024.md:21]: Subsubheading

**Pull request merged**